### DISCUSSION TITLE
HOT RELOAD On Microsoft Visual Studio Professional 2022 (ARM 64-bit) - Preview Version 17.8.0 Preview 5.0

### DISCUSSION THREAD

**Discussion by cgp1976** [dotnet/maui#18327] [General]
Hi,

Is it just me or this feature is totally not working as intended? it use to work in Preview Version 17.8.0 Preview 2.0. are you guys facing the same issues? Basically, modifying the xaml at runtime no longer reflect the changes anymore. I have even try to create a new project (Target Window Platform) from start and also not working.  

any advise?
CGP.

**Comment by Xyncgas** [comment]
I never use hot reload and I made it this far
